contrib/ohi-release-notes/action.yml

```diff
@@ -53,13 +56,23 @@ runs:
       uses: newrelic/release-toolkit/validate-markdown@v1
       with:
         markdown: ${{ inputs.git-root }}/CHANGELOG.md
+    # excluded-dependencies is a file contained by this action and is located inside $GITHUB_ACTION_PATH, but this
+    # folder is not mounted in the container that runs the generate-yaml action, so we copied the file to
+    # ${GITHUB_WORKSPACE} which is mounted in the container.
+    - name: Copy dependencies file to temp folder mounted by the generate-yaml action
+      shell: bash
+      run: |
+        if ! [ -f "${GITHUB_WORKSPACE}/${{ inputs.excluded-dependencies-manifest }}" ]; then
+          cp ${{ github.action_path }}/excluded-dependencies.yml "${GITHUB_WORKSPACE}/${{ inputs.excluded-dependencies-manifest }}"
+        fi
     - name: Generate YAML
-      uses: newrelic/release-toolkit/generate-yaml@v1
+      uses: newrelic/release-toolkit/generate-yaml@msanmiquel/exclude-dev-dependencies
```

Comment:
This is still pointing to the branch.

Reply:
I'm updating it at the end as I was "playing around" 😢   with the self test.